### PR TITLE
Document overriding $PATH with user.conf

### PR DESCRIPTION
**Pull request by snakeroot**
This is per @poettering suggestion in Issue #733

### CODE REVIEW

File: man/systemd.exec.xml [hunk 1]
```diff
@@ -1160,6 +1160,12 @@
         <listitem><para>Colon-separated list of directories to use
         when launching executables. Systemd uses a fixed value of
         <filename>/usr/local/sbin</filename>:<filename>/usr/local/bin</filename>:<filename>/usr/sbin</filename>:<filename>/usr/bin</filename>:<filename>/sbin</filename>:<filename>/bin</filename>.
+        As with all environment variables, <varname>$PATH</varname> can be 
+        overridden using <varname>"DefaultEnvironment="</varname> in the
```
Comment (zonque) [inline man/systemd.exec.xml]: Quotes are not needed inside `<varname>` tags.

File: man/systemd.exec.xml [hunk 2]
```diff
@@ -1160,6 +1160,12 @@
         <listitem><para>Colon-separated list of directories to use
         when launching executables. Systemd uses a fixed value of
         <filename>/usr/local/sbin</filename>:<filename>/usr/local/bin</filename>:<filename>/usr/sbin</filename>:<filename>/usr/bin</filename>:<filename>/sbin</filename>:<filename>/bin</filename>.
+        As with all environment variables, <varname>$PATH</varname> can be 
+        overridden using <varname>"DefaultEnvironment="</varname> in the 
+        <filename>systemd.conf</filename> file for the system instance and in 
+        the relevant <filename>user.conf</filename> file for all user instances 
+        or an individual user instance. See 
+        <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
```
Comment (zonque) [inline man/systemd.exec.xml:1168]: The construction "the ... file" is redundant.

**Comment by poettering** [comment]
THis commit set is broked. Please rebase all commits into one.

**Comment by snakeroot** [comment]
Sorry

**Pull request closed without merging**